SDBlockDevice.cpp

```diff
@@ -147,6 +147,11 @@
 #include "mbed_debug.h"
 #include <errno.h>
 
+/* Required version: 5.5.4 and above */
+#if (MBED_VERSION < MBED_ENCODE_VERSION(5,5,4))
```

Comment:
I can have this macro limited to release (just like mbed-os) but even with that module release will have issues with OS master.
I will add the check for MBED_VERSION and if not defined, will set warning (to verify version) instead macro. Does that sound good enough? In case of master, user will have to make sure that proper combination is picked.